src/main/resources/templates/showProduct.html

```diff
@@ -84,10 +84,18 @@ <h2 class="accordion-header">
                                 <div th:each="i : ${#numbers.sequence(1, review.rating.getValue())}" class="displayStarFill">★</div>
                             </div>
                         </button>
+
                     </h2>
                     <div th:id="${review.reviewID}" class="accordion-collapse collapse ghostwhite-bg-col" data-bs-parent="#reviewAccordion">
                         <div class="accordion-body text-break">
-                            <a th:text="${review?.account?.username}" th:href="'/account/' + ${review?.account?.username}"></a>
+                            <a th:text="${review?.account?.username}" th:href="'/account/' + ${review?.account?.username}" class="h6"></a>
+                            <small>
+                                <div th:switch="${review.account.getSeparation(currentAccount)}" class="text-muted mb-2">
+                                    <div th:case="1">1 degree</div>
```

Comment:
As a user I wouldn't know what just "1 degree" means so maybe instead maybe we can follow how LinkedIn does it (i.e 1st, 2nd, 3rd). Also it should be shown in a user's account page as well.

Reply:
I'd argue both don't make sense if you don't know what the number means. At least degree says the word.

Reply:
Degree now shows on the user's page (user1 visiting user5). Same as before, if degree isn't relevant it wont be shown on the page. 
![image](https://github.com/user-attachments/assets/2edd9bd5-3ef6-43e6-bb7c-3fe80e824100)

- As far as 1 degree vs 1st, neither convey the idea clearly. The only way to clearly convey the idea would be '1st degree connection' which would look like images below. 
- Alternatively, I can do something in the middle like '1st degree' (most recent push). Let me know if you have a preference.   
 
![image](https://github.com/user-attachments/assets/f77a786f-3763-445f-a018-a787852cf55e)
![image](https://github.com/user-attachments/assets/435b25e7-443d-49dc-b400-795c01d9e5b2)